config/deploy.rb

```diff
@@ -30,7 +30,15 @@
 
 # Default value for default_env is {}
 # set :default_env, { path: "/opt/ruby/bin:$PATH" }
-# set :rvm, '2.0.0-p247'
+# set :rvm, '2.2.2'
+
+# rbenv
+set :rbenv_type, :user # or :system, depends on your rbenv setup
+set :rbenv_ruby, File.read('.ruby-version').strip
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.